### PR TITLE
Move docker information

### PR DESCRIPTION
**Pull request by aerokite**
Lets say,
xdb version: `x.y.x`

So,
database image: `kubedb/xdb:x.y.z`
##### For backup/restore purpose
tools image:           `kubedb/xdb-tools:x.y.z`

And

Operator Image: `kubedb/<db code>:tag`

### CODE REVIEW

File: pkg/docker/images.go [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
-	ImageElasticdump       = "kubedb/elasticdump"
-	ImageMongoDBOperator   = "kubedb/mongodb-operator"
-	ImageMongoDB           = "library/mongo"
-	ImageRedisOperator     = "kubedb/redis-operator"
-	ImageRedis             = "library/redis"
-	ImageMemcachedOperator = "kubedb/mc-operator"
-	ImageMemcached         = "library/memcached"
+	ImageOperator              = "kubedb/operator"
+	ImagePostgresOperator      = "kubedb/pg-operator"
+	ImagePostgres              = "kubedb/postgres"
```
Comment (the-redback) [inline pkg/docker/images.go]: Isn't it better this way?
```
ImageOperator              = "/operator"
ImagePostgresOperator      = "/pg-operator"
ImagePostgres              = "/postgres"
```
Then we can simply add docker-registry.

Comment (tamalsaha) [reply pkg/docker/images.go]: :+1:

Comment (aerokite) [reply pkg/docker/images.go]: What about this?
```
ImageOperator              = "operator"
ImagePostgresOperator      = "pg-operator"
ImagePostgres              = "postgres"
```

We can keep here just image name

**Comment by aerokite** [comment]
Lets move this Docker Image information to corresponding Operaotr

**Pull request merged**